### DISCUSSION TITLE
Support iOS11

### DISCUSSION THREAD

**Discussion by fofewyoung** [microsoft/msquic#2867] [Q&A]
i tried to build static library for iOS,  and found it support iOS13 only , i expect it support iOS11 (in Unity Games)
so i changed  DEPLOYMENT_TARGET to 11.0,  and run the build script again, it show: 
platform_posix.c:450:5: error: 'timespec_get' is only available on iOS 13.0 or newer
certificates_darwin.c:152:14: error: 'SecTrustEvaluateWithError' is only available on iOS 12.0 or newer
will msquic support iOS11?

**Comment by nibanks** [comment]
If you have possible fixes, please propose a PR. We don't have a lot of experience with iOS ourselves.

**Reply by fofewyoung** [reply]
> 
i proposed a PR:
https://github.com/microsoft/msquic/pull/2898/files
